front/src/applications/operationalStudies/hooks/useAutoUpdateProjection.ts

```diff
@@ -74,9 +84,9 @@ const useAutoUpdateProjection = (
         dispatch(updateSelectedTrainId(firstValidTrain.id));
       }
       if (isPacedTrain(firstValidTrain.id)) {
-        const editoastPacedTrainId = formatPacedTrainIdToEditoastTrainId(firstValidTrain.id);
-        const occurrenceIdToSelect = formatEditoastTrainIdToOccurrenceId({
-          pacedTrainId: editoastPacedTrainId,
+        dispatch(updateTrainIdUsedForProjection(firstValidTrain.id));
```

Comment:
You can simplify the code after the `if (firstValidTrain)`:
- you can dispatch the trainIdUsedForProjection
- compute the new selected train (depending on if the train is a paced train or not)
- dispatch the new selected train Id

Reply:
done